chunky/src/java/se/llbit/chunky/renderer/scene/Scene.java

```diff
@@ -2559,6 +2566,10 @@ public synchronized void importFromJson(JsonObject json) {
         actors.add(entity);
       }
     }
+
+    octreeImplementation = Octree.ImplementationEnum.values()[
+      json.get("octreeImplementation").asInt(Octree.ImplementationEnum.PACKED.ordinal())
```

Comment:
See above. Also remember to log a warning if the octreeImplementation from the scene file is unknown and fall back to one known implementation. As it is, it would crash if the user puts in invalid values or opens a newer scene file with a future octree format with an older chunky version.

Reply:
Done. As i have implemented, the scene will hold the unknown/invalid value and, at construction, the octree will warn and fallback